translate/storage/placeables/general.py

```diff
@@ -211,7 +211,7 @@ class PunctuationPlaceable(Ph):
     # FIXME this should really be a list created as being the inverse of what
     # is available on the translators keyboard.  Or easily expanded by their
     # configuration.
-    regex = re.compile(ur'''([™©®]|          # Marks
+    regex = re.compile(u'''([™©®]|          # Marks
```

Comment:
Since a char is being removed here the remaining lines of the strings are no longer aligned. Please remove a whitespace on each of the following lines to keep this aligned.